dist/Filter-Simple/Makefile.PL

```diff
@@ -0,0 +1,16 @@
+use strict;
```

Comment:
This should probably have require 5.006, since the module code uses `our`, which eliminates the need to test for ABSTRACT_FROM below.

Reply:
Fixed. Also standardized the INSTALLDIRS and whitespace layout.